supertokens_python/recipe/session/asyncio/__init__.py

```diff
@@ -49,12 +49,29 @@ async def create_new_session(
 ) -> SessionContainer:
     if user_context is None:
         user_context = {}
+    if session_data is None:
+        session_data = {}
+    if access_token_payload is None:
+        access_token_payload = {}
+
+    claims_added_by_other_recipes = SessionRecipe.get_claims_added_by_other_recipes()
```

Comment:
`get_claims_added_by_other_recipes` should not be a static function - see node PR.

Reply:
Done